### DISCUSSION TITLE
Implement support to upload multiple Files

### DISCUSSION THREAD

**Discussion by ps-carvalho** [hydrostack/hydro#88] [Ideas]
Congratulations on a great project first of all and very happy to support you.
I think support for multiple Files uploads needs to be implemented as soon as possible.
I'm happy to contribute to this particular feature. 
thank you in advance.
Paulo

**Comment by kjeske** [comment]
Thanks! And thank you very much for your support! ♥️

Yes, sending multiple files sounds good as a improvement to prioritize. In the code I'm taking only the first one:
```
formData.set(propertyName, el.files[0]);
```

Would need to check what happens when you send all of them and your model property is a list.

**Comment by kjeske** [comment]
Moving to issue https://github.com/hydrostack/hydro/issues/92

**Comment by kjeske** [comment]
Fixed in [v0.16.2](https://github.com/hydrostack/hydro/releases/tag/0.16.2)
